baybe/simulation.py

```diff
@@ -446,10 +447,11 @@ def simulate_experiment(
           The individual measurements obtained for the respective target and iteration
     """
     # TODO: Due to the "..." operator, sphinx does not render this properly. Might
-    # want to investigate in the future.
-    # TODO: In the markdown variant, bullet points with more levels are not rendered
-    # properly. We thus might want to refactor this when using html based documentation
-    # Validate the lookup mechanism
+    #   want to investigate in the future.
+    # TODO: Use a `will_terminate` campaign property to decide if the campaign will
```

Comment:
question: is this really necessary? what would you use it for? The solution below of stopping the simulation a few iterations before 100% of points are measures is totally fine. Is there any reason why all simulations MUST go to 100%  (if `n_doe_iterations` was `None`)?